### DISCUSSION TITLE
如何使用 GitHub 的 Actions 构建 Doris ？

### DISCUSSION THREAD

**Discussion by JaneConan** [apache/doris#24612] [A - General / Q&A] [answered]
如何使用 GitHub 的 Actions 构建 Doris ？ 
本地的环境，不同的分支会有不同的异常

**Comment by yagagagaga** [comment]
You can refer to the [official documentation of GitHub action](https://docs.github.com/en/actions/quickstart).

**Reply by JaneConan** [reply]
bt，we see this repo alse have the Doris CI, and it doesn't work... so we just want to CI the be and fe. Could u show me the sample, thx!

**Reply by JaneConan** [reply]
or we can make this repo CI work, because build dependent on Docker image, this action we cannot publish by ourselves. THX

**Reply by yagagagaga** [reply]
You can disable GitHub Action of your Doris fork via https://github.com/JaneConan/doris/settings/actions . 
<img width="1231" alt="image" src="https://github.com/apache/doris/assets/21151126/d41cb49a-df01-4ff2-9b5b-bc72c33745a7">

**Reply by yagagagaga** [reply]
> or we can make this repo CI work, because build dependent on Docker image, this action we cannot publish by ourselves. THX

Or you can delete files under `.github` folder and upload your action file.

**Reply by JaneConan** [reply]
Yes, we know, I means we can make one right and work CI Action with this repo. we doesn't know how to use the Doris-Build-Docker-image in the GitHub Action, we need your help.THX

**Reply by JaneConan** [reply]
就是我想说，咱们本地构建太慢了...我们想托管在 GitHub Action 上，然后想请您们提供一下支持，谢谢

**Reply by yagagagaga** [reply, marked as answer]
The machine built by Github Action uses a virtual machine with the following configuration:
- 2 core CPUs
- 7 GB of RAM memory
- 14 GB of SSD disk space

Obviously, if you think that local compilation is too slow, then the machine provided by GitHub action cannot meet your needs. You need to purchase a lot high performance machine and bind it to the runner of GitHub action.
